user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -839,6 +840,23 @@ def __build_global_report_summary(self,
                                                                                'clusterShapeRecommendation', 'path'))
         unsupported_ops_obj = UnsupportedOpsStageDuration(self.ctxt.get_value('local', 'output',
                                                                               'unsupportedOperators'))
+        rapids_output_dir = self.ctxt.get_rapids_output_folder()
+        unsupported_ops_file_path = FSUtil.build_path(rapids_output_dir,
```

Comment:
Thanks @parthosa for the review. The reason I am passing the file_path instead of  `unsupported_ops_df` is because if we want to run the stats file independently then the input would be file paths of unsupportedOperators.csv, stages.csv and output file. We can sync offline on your refactor approach and evaluate how to change this.